File: Perl/count_word.pl
#!/usr/bin/perl -w

$word = $ARGV[0];

chomp(@lines = <STDIN>);

$nb_of_word = 0;

foreach $line(@lines){
	@lst = split /[^a-zA-Z]/, $line;
	foreach $var(@lst){
		$var =~ tr/A-Z/a-z/;
		if($var eq $word){
			$nb_of_word++;
		}
	}
}

print "$word occurred $nb_of_word times\n";

File: Python/count_word.py
#!/usr/bin/python3

import sys
import re

target_word = sys.argv[1]
nb_of_target_word = 0

for line in sys.stdin:
    lst = re.findall(r'[a-zA-Z]+', line)
    lst = [i.lower() for i in lst]
    nb_of_target_word += lst.count(target_word)

print('{} occurred {} times'.format(target_word, nb_of_target_word))

